nomad/serf.go

```diff
@@ -127,9 +132,22 @@ func (s *Server) maybeBootstrap() {
 	}
 
 	// Update the peer set
-	s.logger.Printf("[INFO] nomad: Attempting bootstrap with nodes: %v", addrs)
-	if err := s.raft.SetPeers(addrs).Error(); err != nil {
-		s.logger.Printf("[ERR] nomad: failed to bootstrap peers: %v", err)
+	// Attempt a live bootstrap!
```

Comment:
In a later commit I ended up adding an additional check here. This isn't strictly required, but helps prevent spurious leader elections if another server is added later - https://github.com/hashicorp/consul/pull/2319.